k8s/demo/common/money-claims/v1.yaml

```diff
@@ -0,0 +1,188 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: demo-v1
+  namespace: money-claims
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: demo-v1
+  chart:
+    repository: https://hmcts.azurecr.io/helm/v1/repo/
+    name: chart-cmc
+    version: 0.0.1
+  values:
+    ### CMC SERVICES ###
+    cmc-claim-store:
+      java:
+        releaseNameOverride: demo-v1-cmc-claimstore
+        secrets: 
+          GOV_NOTIFY_API_KEY:
+              secretRef: cmc-demo-notify-api-key
+              key: key
+          OAUTH2_CLIENT_SECRET:
+              secretRef: cmc-demo-oauth-client-secret
+              key: key
+          IDAM_CASEWORKER_ANONYMOUS_USERNAME:
+              secretRef: cmc-demo-anonymous-caseworker-username
+              key: key
+          IDAM_CASEWORKER_ANONYMOUS_PASSWORD:
+              secretRef: cmc-demo-anonymous-caseworker-password
+              key: key
+          IDAM_CASEWORKER_SYSTEM_USERNAME:
+              secretRef: cmc-demo-system-update-username
+              key: key
+          IDAM_CASEWORKER_SYSTEM_PASSWORD:
+              secretRef: cmc-demo-system-update-password
+              key: key
+        environment:
+          SPRING_MAIL_HOST: mta.reform.hmcts.net 
```

Comment:
this probably won't work, it's not peered there

Reply:
can i use mailhog? then add as a dep to chart-cmc? ref: https://github.com/codecentric/helm-charts - just copy paste chart into hmcts chart-mailhog with azure pipeline. or fork and add our build bits.

Reply:
you can use it, you could probably just deploy one mail hog for all of your demos, with a manifest even if you want....

Reply:
cool - im going to do this first, in separate PR 👍